libgalois/include/galois/OpLog.h

```diff
@@ -19,6 +19,19 @@ enum class OpTypes {
   kOpEdgePropVal,
 };
 
+const char* const optypes_enum2str[] = {
+    "Invalid",     /* kInvalid */
+    "NodeAdd",     /* kOpNodeAdd */
+    "NodeDel",     /* kOpNodeDel */
+    "EdgeAdd",     /* kOpEdgeAdd */
+    "EdgeDel",     /* kOpEdgeDel */
+    "NodePropDel", /* kOpNodePropDel */
+    "EdgePropDel", /* kOpEdgePropDel */
+    "NodePropVal", /* kOpNodePropVal */
+    "EdgePropVal", /* kOpEdgePropVal */
+    "Consumed",    /* kConsumed */
+};
+
```

Comment:
comment on the lines below: SetNProp => SetNodeProp. Most of the other code writes "node" and "edge" out, though this is more important for public symbols.

Reply:
This is not from this PR. It's some old code from @witchel .

Reply:
I don't think these names are (easily) public so it doesn't bother me, but this is something to consider during future iterations.

Reply:
That name was my bad.  I was just seeing how it felt to shorten the identifiers.

In general, I think it is more straightforward to just spell out node/edge and have separate functions for them even though the temptation to have a single algorithm that can deal with both is powerful.

Reply:
I feel your pain.

I created this widget in PropertyFileGraph https://github.com/KatanaGraph/katana/blob/54da9508373e093a1184955fa23d5dbe12e0d7bd/libgalois/include/galois/graphs/PropertyFileGraph.h#L270
to allow library code to reuse as much between edges and nodes while still presenting separately named edge and node functions to the end-user.

Reply:
When collecting a set of pointers to member functions one must ask, "Am I a force for good?" 😎